### PR TITLE
dog.0.1.0 - via opam-publish

### PR DESCRIPTION
**Pull request by samoht**
A loyal and faithful synchronisation tool that you can rely on.

---
* Homepage: https://github.com/samoht/dog
* Source repo: https://github.com/samoht/dog.git
* Bug tracker: https://github.com/samoht/dog/issues

---
Pull-request generated by opam-publish v0.2.1

### CODE REVIEW

**Comment by samoht** [comment]
Hum something is wrong in the constraints:
> Skipping dog.0.1.0 as not installable

**Comment by avsm** [comment]
Fairly unhelpful travis error:

```
#=== ERROR while installing dog.0.1.0 =========================================#
# opam-version 1.2.0
# os           linux
# command      ./configure --prefix /home/travis/.opam/system --{alcotest:enable}-tests
# path         /home/travis/.opam/system/build/dog.0.1.0
# compiler     system (4.02.1)
# exit-code    1
# env-file     /home/travis/.opam/system/build/dog.0.1.0/dog-26477-82479e.env
# stdout-file  /home/travis/.opam/system/build/dog.0.1.0/dog-26477-82479e.out
# stderr-file  /home/travis/.opam/system/build/dog.0.1.0/dog-26477-82479e.err
### stderr ###
# ...[truncated]
#   --sysconfdir dir     Read-only single-machine data [$prefix/etc]
#   --libexecdir dir     Program executables [$exec_prefix/libexec]
#   --sbindir dir        System admin executables [$exec_prefix/sbin]
#   --bindir dir         User executables [$exec_prefix/bin]
#   --exec-prefix dir    Install architecture-dependent files in dir [$prefix]
#   --prefix dir         Install architecture-independent files dir [/usr/local]
#   --print-hidden Print even non-printable variable. (debug)  [false]
#   -help                Display this list of options
#   --help               Display this list of options
# 
=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
```

File: packages/dog/dog.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  [make]
+]
+build-test: [
+  [make "test"]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "dog"]
+  ["rm" "-f" "%{bin}%/dog"]
+]
```
Comment (avsm) [inline packages/dog/dog.0.1.0/opam]: why not a `dog.install` file for this?

**Comment by samoht** [comment]
Very weird error message in travis where `%{lwt:enable}%` is `disable` even if `lwt` is installed. @AltGr any idea? (see https://travis-ci.org/ocaml/opam-repository/jobs/50182523, it seems to happens only for 1.1)

File: packages/dog/dog.0.1.0/opam (hunk 1 above)
Comment (samoht) [inline packages/dog/dog.0.1.0/opam]: cause I'm too lazy to enumerate all the library files to install :p I'll check if `oasis2opam` can help me.

**Comment by avsm** [comment]
but opam 1.2 fails too...

**Comment by avsm** [comment]
Needs a 4.02.1 constraint and it can be merged.

**Pull request merged**